hub/util/no_view.py

```diff
@@ -0,0 +1,18 @@
+from hub.util.exceptions import InvalidOperationError
+from typing import Callable
+from functools import wraps
+import hub
+
+
+def no_view(callable: Callable):
```

Comment:
This is cool.  Although, the name is not descriptive for me.  Seems like this decorator is checking for something and the name should reflect that?